FABulous/fabric_definition/Tile.py

```diff
@@ -63,6 +69,10 @@ def __init__(
         for b in self.bels:
             self.globalConfigBits += b.configBit
 
+        for gio in self.gen_ios:
+            if gio.configAccess:
+                self.globalConfigBits += gio.configBit
```

Comment:
It might lead to a problem. Since if we use Gen IO after tile creation, this will not be updated. We should make the global config bit a class attribute, and compute the result every time this is used. Less efficient, but always right. 

Something like:
```python
@property
def globalConfigBits() -> int:
  configBit = 0
  configBit += sum([...]) #bel sum
  configBit += sum(...) #switch matrix sum
  return configBit
  
```

Reply:
Good idea, I'll implement this. :)

Reply:
https://github.com/FPGA-Research/FABulous/pull/380/commits/50882fa23e9799384fb40b2a2f977b6fb726f364

Reply:
https://github.com/FPGA-Research/FABulous/pull/380/commits/571f0e1a06ac672b208425edd33f9be54bd04c0f

Since we add the IO Bel to the tile after generating it, I don't need to count the genIO config bits manually anymore. :)